packages/cli/src/pages/api/messages/index.ts

```diff
@@ -15,7 +17,94 @@ export default async function handler(
     return res.status(405).json({ error: "Method not allowed" });
   }
 
-  if (!(await validateApiKey(req, res))) return;
+  const apiKey = apiKeyFromReq(req);
```

Comment:
should write some tests for this file.  this logic look right though?

Reply:
yeah looks right. one little error I found.